src/components/lightbox/lightbox.js

```diff
@@ -0,0 +1,65 @@
+import mixin from '../../globals/js/misc/mixin';
+import createComponent from '../../globals/js/mixins/create-component';
+import initComponentBySearch from '../../globals/js/mixins/init-component-by-search';
+import eventMatches from '../../globals/js/misc/event-matches';
+
+class Lightbox extends mixin(createComponent, initComponentBySearch) {
+  constructor(element, options) {
+    super(element, options);
+    this.activeIndex = this.element.dataset.lightboxIndex;
+    this.totalSlides =
+      [...this.element.querySelectorAll(this.options.selectorLightboxItem)].length - 1;
+
+    this.updateSlide();
+
+    this.element.addEventListener('click', evt => this.handleClick(evt));
+
+    this.element.parentNode.addEventListener('modal-beingshown', evt => {
+      if (!evt.detail.launchingElement.dataset.carouselItemIndex) {
+        throw new Error(
+          'launchingElement must have carouselItemIndex data attribute to indicated what item to display'
+        );
+      }
+      this.activeIndex = evt.detail.launchingElement.dataset.carouselItemIndex;
+      this.updateSlide();
+    });
+  }
+
+  handleClick = evt => {
+    if (evt.target.matches(this.options.selectorScrollRight)) {
+      if (this.activeIndex < this.totalSlides) {
+        this.activeIndex++;
+        this.updateSlide();
+      }
+    }
+
+    if (evt.target.matches(this.options.selectorScrollLeft)) {
+      if (this.activeIndex > 0) {
+        this.activeIndex--;
+        this.updateSlide();
+      }
+    }
+  };
+
+  updateSlide = () => {
+    const items = [...this.element.querySelectorAll(this.options.selectorLightboxItem)];
+    items.forEach(item => item.classList.remove(this.options.classActiveItem));
+    items[this.activeIndex].classList.add(this.options.classActiveItem);
```

Comment:
Given `this.activeIndex` seems to initially come from `data-carousel-item-index` - Should we check if `this.activeIndex` is negative or larger than (or equal to) `items.length` and throw an `RangeError`?